### PR TITLE
add hougangliu as a reviewer

### PR DESCRIPTION
**Pull request by hougangliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/310)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -9,3 +9,4 @@ reviewers:
     - libbyandhelen
```
Comment (jlewi) [inline OWNERS]: Can you fix the indentation; these items should not be indented relative to jose5918.

**Comment by hougangliu** [comment]
@jlewi It is very strange that all reviewers keep same indentation in my local repo, but in GitHub review page it looks these items are indented relative to jose5918.
When I cut down one indentation for all items. It seems that GitHub review page show rightly

**Comment by jlewi** [comment]
Is there a mix of tabs and spaces?

**Comment by hougangliu** [comment]
@jlewi I verified that all indentations are spaces. When I `dd` the first two lines and `yy` in the last line as change set. github page shows well now.
Maybe you can git clone and verify it if you're interested :)

**Pull request merged**